com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -850,7 +828,7 @@ internal static void HandleNetworkVariableUpdate(List<INetworkVariable> networkV
 
                     long readStartPos = stream.Position;
 
-                    networkVariableList[i].ReadField(stream, NetworkTickSystem.NoTick, NetworkTickSystem.NoTick);
+                    networkVariableList[i].ReadField(stream, NetworkTimeSystem.NoTick); // TODO broken this use the correct tick
```

Comment:
Anyone knows why this is using NoTick? Looks like this is broken?

Reply:
Looks like the tick parameters are ignored in all impl's of ReadField() (e.g. NetworkDictionary, NetworkList, NetworkSet) with the only exception being that NetworkVariable forwards it to its LocalTick & RemoteTick variables.  

LocalTick itself is never used except in `ManualNetworkVariableTest.cs`, which we could re-write.

RemoteTick is written to the network in NetworkBehaviour::NetworkVariableUpdate with the comment "this will allow lag-compensation".

Finally, this gets read in at NetworkBehaviour::HandleNetworkVariableDeltas, and shoved back down ReadDelta, where it is never actually used.

I sure could be missing something, but I think this may all be no op code and we can delete this parameter.

Reply:
For your consideration, https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/853

Reply:
I don't think we should just remove this. As mentioned in my comment in #853 this was just recently added in anticipation of the snapshot work? If the snapshot system will make this work differently then please lets refactor this or come up with a better design instead of scrapping it first?